src/libraries/Microsoft.Extensions.Caching.Memory/tests/TokenExpirationTests.cs

```diff
@@ -162,6 +162,30 @@ public void RemoveItemDisposesTokenRegistration()
             Assert.True(callbackInvoked.WaitOne(TimeSpan.FromSeconds(30)), "Callback");
         }
 
+        [Fact]
+        public void ClearingCacheDisposesTokenRegistration()
+        {
+            var cache = (MemoryCache)CreateCache();
+            string key = "myKey";
+            var value = new object();
+            var callbackInvoked = new ManualResetEvent(false);
+            var expirationToken = new TestExpirationToken() { ActiveChangeCallbacks = true };
+            cache.Set(key, value, new MemoryCacheEntryOptions()
+                .AddExpirationToken(expirationToken)
+                .RegisterPostEvictionCallback((subkey, subValue, reason, state) =>
+                {
+                    // TODO: Verify params
```

Comment:
What's the plan to address this TODO?

Reply:
It's a TODO I have copy-pasted from other test that I've used as base for the new test. I am going to remove it.

Reply:
```suggestion
```